fdbserver/RocksDBLogForwarder.actor.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * RocksDBLogForwarder.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbserver/RocksDBLogForwarder.h"
+
+#include "flow/network.h"
+#include "flow/Trace.h"
+#include "fdbrpc/simulator.h"
+
+#include "flow/actorcompiler.h" // This must be the last include file
+
+using InfoLogLevel = rocksdb::InfoLogLevel;
+
+namespace {
+
+Severity getSeverityFromLogLevel(const InfoLogLevel& log_level) {
+	switch (log_level) {
+	case InfoLogLevel::DEBUG_LEVEL:
+		return SevDebug;
+	case InfoLogLevel::INFO_LEVEL:
```

Comment:
If this log level is too spammy. Lets try to add a knob for the minimum log level to print.

Reply:
I think at this stage, we could try to log at INFO level. Since we will need to revisit the log levels later, as some of RocksDB fatals are not FDB fatals, and should not trigger SevError.